### ISSUE TITLE
[1.18.2] [Client] Game will crash whenever I directly look at grass, using grasskiss mod alongside lithium.

### ISSUE THREAD

**Issue by akai-hana** [jellysquid3/Lithium#378]
### Version
> lithium-fabric-mc1.18.2-0.7.10.jar

### Reproduction Steps
> When pairing this version of lithium with grasskiss mod (grasskiss-1.18.2-fabric0.47.10-0.5.0.jar), you'll get a crash if you look at grass (A very strange bug, I have to say).

### Crash Report File
[crash-2022-07-23_14.56.40-client.txt](https://github.com/CaffeineMC/lithium-fabric/files/9173839/crash-2022-07-23_14.56.40-client.txt)

### Minecraft Save
> Any save works. Just create a new world and place grass, or look at already generated grass, you'll get a crash anyways.

### Additional Information

> When I remove either grasskiss or lithium, it'll stop crashing when I look at grass. So there's that confirmation.

**Comment by Indignified** [comment]
yeah what i've found is coffee client doesn't mix well with lithium so you probably have to run it without lithium

**Comment by 2No2Name** [comment]
add `mixin.shapes.lazy_shape_context=false` to the lithium.config file

**Comment by 2No2Name** [comment]
https://github.com/CaffeineMC/lithium-fabric/commit/33ea83fd6a08fd2120b5a290c77a1c4fd53145a3
This should allow fixing this properly by writing a method called getHeldItem(), lithium will mixin into it and initialize the item
